### DISCUSSION TITLE
Select Artboards on other pages

### DISCUSSION THREAD

**Discussion by neodymium3** [uxbox/uxbox#960] [Ideas]
It seems like there should be a way to select artboards on other pages within the project when using the Prototype feature. Some elements should link to what would be a different page, such as having a home button return to the first page.

It would be nice when you select an element and use the "Navigate to" drop down to see something like "Page-1: Artboard-1", instead of only artboards on the same page.

Not being able to do this encourages putting everything on a single page and having tons of artboards on it so that for any element any other artboard can be chosen.

Outside of that, this is a really neat tool and is fun to use. Thank you!

**Comment by hirunatan** [comment]
This is a complex issue, because it may have a lot of side effects. But also it's true that the current way forces you to have all artboards in the same page, and this is not very good.

We have added an enhancement to think about it: https://tree.taiga.io/project/penpot/issue/1704

**Comment by lemonkiller** [comment]
This will make teamwork and large projects much smoother! Definitely a feature worth promoting.
